cfpb/consumerfinance.gov
# Retirement: Move analytics tracking code inside formSubmitted event handler

## Changes

- Move tracking code inside formSubmitted event handler.
- Update test to wait for URL on Spanish page to load.


## How to test this PR

1. Pull branch, `./frontend.sh`
2. Visit http://localhost:8000/consumer-tools/retirement/before-you-claim/?debug-gtm=true and there should be tracking events logged to the dev console when interacting with the retirement tool.
